pkg/ucp/store/apiserverstore/apiserverclient_test.go

```diff
@@ -225,79 +225,79 @@ func Test_APIServer_Client(t *testing.T) {
 
 		// In this test we're going to **similuate** a hash collision and verify that it is saved correctly.
 		//
-		// Let's PRETEND that shared.BasicResource1ID and shared.BasicResource2ID result in the same
+		// Let's PRETEND that shared.BasicResource2ID and shared.BasicResource3ID result in the same
```

Comment:
This test needed a rework because it relied on a "multi-type query". I made changes here so that it's still covering the same case, but with two resources of the same type.